it_zwierzak/adoption_announcement/models.py

```diff
@@ -16,13 +17,42 @@ class Announcement(models.Model):
     def __str__(self):
         return f'{self.species} {self.name} - {self.age} lat'
 
-class Member(models.Model):
-    name = models.CharField(max_length=20)
-    surname = models.CharField(max_length=30)
-    birth_date = models.DateField()
-    email = models.EmailField(max_length=50)
-    phone_number = models.IntegerField()
-    password = models.CharField(max_length=50)
+class AdoptionApplication(models.Model):
+    your_education = models.CharField(max_length=250)
 
-    def __str__(self):
-        return f'{self.name} {self.surname}'
+    number_of_adults_in_the_house = models.PositiveSmallIntegerField(default=1, validators=[
+        MinValueValidator(1),
+        MaxValueValidator(10),
+    ])
+    # number_of_children_in_the_house = models.CharField(choices=((0, 0), (1, 1), (2, 2), (3, 3), (4, 4), (5, 5), (6, 6), (7, 7), (8, 8), (9, 9), (10, 10)), max_length=1, default=0)
+
+    number_of_children_in_the_house = models.PositiveSmallIntegerField(default=0, validators=[
+        MinValueValidator(0),
+        MaxValueValidator(10),
+    ])
+
+    other_animals = models.CharField(max_length=250)
+
+    hours_away_from_home = models.PositiveSmallIntegerField(default=0, validators=[
+        MinValueValidator(0),
+        MaxValueValidator(12),
+    ])
+
+    what_about_moving = models.CharField(max_length=300)
```

Comment:
what about it? Nazwa trochę dziwna, `will_be_moving` i w help text opisać to trochę, albo wywalić całkiem, też max length powyżej 255 to zmieniłbym na TextField

Reply:
moving_out, charfield 250